_posts/en/newsletters/2019-08-07-newsletter.md

```diff
@@ -0,0 +1,148 @@
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to Bitcoin Core 0.18.1:** this maintenance release makes
+  available several bug fixes and other improvements.  [Upgrading][bitcoin
+  core 0.18.1] is recommended.
+
+## News
+
+- **BIP174 extensibility:** the author of the Partially-Signed Bitcoin
+  Transactions (PSBTs) specification, Andrew Chow, [proposed][psbt
+  extensions] a few minor changes for general adoption:
+
+    - *Reserved types for proprietary use:* some applications are
+      already including data in PSBTs using types that aren't specified
+      in [BIP174][].  It's proposed that one type byte or a range of
+      type bytes be reserved for private PSBTs extensions, similar to
+      reserved IP address ranges for private networks.  The exact
+      construction of this mechanism was a particular focus for
+      discussion this week.
+
+    - *Global version number:* although the goal is to design enhancements
+      to PSBTs so that they're backwards compatible, Chow proposes to
+      add a version byte to PSBTs indicating the most advanced feature
+      they use so that older parsers can detect when they're being given
+      a PSBT they might not understand.  PSBTs without an explicit
+      version number would be considered to use version 0.
+
+    - *Multi-byte types:* To allow more types, multi-byte types are
+      proposed.  Mailing list discussion seems to favor using the same
+      CompactSize unsigned integers used in the Bitcoin protocol.
+
+- **Trampoline payments:** Bastien Teinturier opened a [PR][trampoline
+  pr] in the BOLTs repository and started a [discussion][trampoline
+  discussion] on the Lightning-Dev mailing list about adding support to
+  the protocol for the trampoline payments described in [Newsletter
+  #40][news40 trampoline payments] where the spender sends the payment
+  to an intermediate node who calculates the path either to another
+  intermediate node (for privacy) or to the receiving node.  This can be
+  very beneficial to low-bandwidth LN clients (such as on mobile
+  devices) that don't want to keep up with gossip traffic and so only
+  know how to route to a small number of nodes.  Sending a trampoline
+  payment will require coordination between multiple nodes and so
+  requires modification of the Lightning specification.
+
+## Bech32 sending support
+
+*Week 21 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% include specials/bech32/21-brd.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #15911][] changes the `walletcreatefundedpsbt` to signal
```

Comment:
Suggestion: s/changes/improves/ to change things up with the changes in the next item

Reply:
Repeated phrasing in paragraph text (such as the "and so" you identified earlier) can indeed be jarring to read and so should be avoided, but in lists, I think it's beneficial as the parallelism makes reading (and particularly skimming) faster.  My usual template for Bitcoin Core RPC changes is to say *adds*, *removes*, or *changes* followed by the RPC name(s) followed by the actual changes.  Since most people only use a few RPCs, that puts the information they need to quickly figure out whether this change affects them near the start of the list item, making skimming more effective.  (As a general rule, I will quite often use somewhat odd phrasing near the start of a list item so that it's easier to skim because it starts by mentioning the systems affected.)